components/advanced/babel-plugin.js

```diff
@@ -0,0 +1,174 @@
+        </Code>{' '}
+        instead of just <Code>&lt;button class=&quot;asdf123&quot; /&gt;</Code>.
+      </p>
+      <p>
+        This also adds support for showing your components' real name in the
+        React DevTools. Consider writing a styled component that renders a{' '}
+        <Code>button</Code> element, called <Code>MyButton</Code>. It will
+        normally show up as <Code>&lt;styled.button&gt;</Code> for all of your
+        components, but with this plugin they show{' '}
+        <Code>&lt;MyButton /&gt;</Code>.
+      </p>
+      <p>
+        This makes it easier to find your components and to figure out where
+        they live in your app.
+      </p>
+      <p>
+        If you don't need this feature, you can disable it with the{' '}
+        <Code>displayName</Code> option:
+      </p>
+      <CodeBlock code={displayName} language="node" />
+    </SectionLayout>
+    <SectionLayout sub title="Preprocessing">
+      <Note>
+        This is experimental and we don't yet know of all limitations and bugs!
+        Consider this non-production ready for now. ⚠️
+      </Note>
+      <p>
+        This plugin preprocesses your styles with stylis and uses the{' '}
+        <Code>no-parser.js</Code> entrypoint on styled-components.
+      </p>
+      <p>
+        This effectively removes stylis from your runtime bundle and should
+        slightly improve runtime performance and shrink your bundle size.
+      </p>
+      <p>
+        It automatically disables the <Code>minify</Code> option, since stylis
+        already does some minification on your CSS.
+      </p>
+      <p>
+        You can enable preprocessing with the <Code>preprocess</Code> option:
+      </p>
+      <CodeBlock code={preprocess} language="node" />
+    </SectionLayout>
+    <SectionLayout sub title="Minification">
+      <Note>
+        This option is turned on by default. If you experience mangled CSS
+        results, turn it off and open an issue please.
+      </Note>
+      <p>
+        This plugin minifies your styles in the tagged template literals, giving
+        you big bundle size savings.
+      </p>
+      <Note>
+        This operation may potentially break your styles in some rare cases, so
+        we recommend to keep this option enabled in development if it's enabled
+        in the production build.
+      </Note>
+      <p>
+        You can disable minification with the <Code>minify</Code> option:
+      </p>
+      <CodeBlock code={minify} language="node" />
+    </SectionLayout>
+    <SectionLayout sub title="Template String Transpilation">
+      <p>
+        We also transpile <Code>styled-components</Code> tagged template
+        literals down to a smaller representation than what Babel normally does,
+        because <Code>styled-components</Code> template literals don't need to
+        be 100% spec compliant. see{' '}
+        <a href="#tagged-template-literals">Tagged Template Literals</a> for
```

Comment:
This link will need to be swapped out with our Link component :wink: You can see some examples for this on other pages